crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/lambda.py

```diff
@@ -75,3 +75,9 @@ def f(
 
 # ...but we do preserve a trailing comma after the arguments
 a = lambda b,: 0
+
+lambda *a: ()
```

Comment:
Unfortunately you can do disturbing things like insert a comment between the star and the parameter. We'll probably want test coverage for cases like this:

```python
(
    lambda
    *
    # test
    x: print(x)
)
```